tornado/websocket.py

```diff
@@ -238,6 +238,16 @@ def get_compression_options(self):
         """
         return None
 
+    def get_handshake_response_headers(self):
```

Comment:
Why does this need to be a new method? Why not override `prepare()` and call `set_header()` there?